### ISSUE TITLE
sandbox launch error

### ISSUE THREAD

**Issue by lotux** [naparuba/shinken#1495]
running sanbox launch as following give me error and doesn't run arbiterd
```
[root@spare3-pc shinken-2.2]# ./bin/launch_all.sh
```

```
TOTAL serializing in 0.0132899284363
[1423075334] INFO: [Shinken] Configuration Loaded
[1423075334] CRITICAL: [Shinken] I got an unrecoverable error. I have to exit.
[1423075334] CRITICAL: [Shinken] You can get help at https://github.com/naparuba/shinken
[1423075334] CRITICAL: [Shinken] If you think this is a bug, create a new ticket including details mentioned in the README
[1423075334] CRITICAL: [Shinken] Back trace of the error: Traceback (most recent call last):
  File "/tmp/shinken-2.2/shinken/daemons/arbiterdaemon.py", line 602, in main
    self.do_daemon_init_and_start()
  File "/tmp/shinken-2.2/shinken/daemon.py", line 609, in do_daemon_init_and_start
    self.check_parallel_run()
  File "/tmp/shinken-2.2/shinken/daemon.py", line 446, in check_parallel_run
    self.__open_pidfile()
  File "/tmp/shinken-2.2/shinken/daemon.py", line 433, in __open_pidfile
    raise InvalidPidFile(err)
InvalidPidFile: [Errno 13] Permission denied: u'/var/run/shinken/arbiterd.pid'

Traceback (most recent call last):
  File "/tmp/shinken-2.2/bin/../bin/shinken-arbiter", line 116, in <module>
    daemon.main()
  File "/tmp/shinken-2.2/shinken/daemons/arbiterdaemon.py", line 602, in main
    self.do_daemon_init_and_start()
  File "/tmp/shinken-2.2/shinken/daemon.py", line 609, in do_daemon_init_and_start
    self.check_parallel_run()
  File "/tmp/shinken-2.2/shinken/daemon.py", line 446, in check_parallel_run
    self.__open_pidfile()
  File "/tmp/shinken-2.2/shinken/daemon.py", line 433, in __open_pidfile
    raise InvalidPidFile(err)
shinken.daemon.InvalidPidFile: [Errno 13] Permission denied: u'/var/run/shinken/arbiterd.pid'
```

all other pid files are created properly, but for some reason arbiterd.pid fails, there is no previous pid file for arbiterd in /var/run/shinken and I ran lanuch script as root

**Comment by titilambert** [comment]
Hello ! 
which version do you use ? Upstream ?

**Comment by titilambert** [comment]
Hello !
This bug should be fixed with shinken 2.4-BETA1. So, I close this issue.
Fell free to reopen it, if you still have this error ;)
